tools/spinel-cli/README.md

```diff
@@ -0,0 +1,1327 @@
+	with a spinel spi-driver daemon.
+	Note: <SOCKET> will eventually map to hostname:port tuple.
+
+    -n NODEID, --nodeid=NODEID
+        The unique nodeid for the HOST and NCP instance.
+
+    -q, --quiet
+        Minimize debug and log output.
+
+    -v, --verbose
+        Maximize debug and log output.
+
+## Quick Start
+
+The spinel-cli tool provides an intuative command line interface, including
+all the standard OpenThread CLI commands, plus full history accessable by 
+pressing the up/down keys, or searchable via ^R.  There are a few commands
+that spinel-cli provides as well that aren't part of the standard set 
+documented in the command reference section.
+
+```
+openthread$ cd tools/spinel-cli/
+spinel-cli$ ./spinel-cli.py
+Opening pipe to ../../examples/apps/ncp/ot-ncp 1
+spinel-cli > help
+
+Available commands (type help <name> for more information):
+============================================================
+channel            extaddr       mode              route
+child              extpanid      netdataregister   router
+childtimeout       h             networkidtimeout  routerupgradethreshold
+clear              help          networkname       scan
+contextreusedelay  history       panid             state
+counter            ifconfig      ping              thread
+debug              ipaddr        prefix            v
+debug-term         keysequence   q                 version
+eidcache           leaderdata    quit              whitelist
+enabled            leaderweight  releaserouterid
+exit               masterkey     rloc16
+
+spinel-cli > version
+OPENTHREAD/gd4d4e9d-dirty; Aug 11 2016 14:40:44
+Done
+spinel-cli > thread start
+Done
+spinel-cli > state
+leader
+Done
+spinel-cli >
+```
+
+## Running the NCP Tests
+
+The OpenThread automated test suite can be run against any of the following 
+node types by passing the NODE_TYPE environment variable:
+
+| NODE_TYPE     |    Description                                     |
+|---------------|----------------------------------------------------|
+| sim (default) | Runs against ot-cli posix emulator                 |
+| ncp-sim       | Runs against ot-ncp posix emulator with spinel-cli |
+| soc           | Runs against CLI firmware on a device connected via /dev/ttyUSB<nodeid>  |
+
+### Manual run of NCP thread-cert test
+
+```
+# From top-level of openthread tree
+./bootstrap
+./configure --with-examples=posix --enable-cli --enable-ncp=uart
+make
+cd tests/scripts/thread-cert
+NODE_TYPE=ncp-sim top_builddir=../../.. python Cert_5_1_02_ChildAddressTimeout.py VERBOSE=1
+```
+
+### Run entire NCP thread-cert suite
+
+```
+# From top-level of openthread tree
+make distclean
+./bootstrap
+NODE_TYPE=ncp-sim BUILD_TARGET=posix-distcheck DISTCHECK_CONFIGURE_FLAGS="--with-examples=posix --enable-cli --enable-ncp=uart --with-tests=all" make -f examples/Makefile-posix distcheck BuildJobs=10 VERBOSE=1
+```
+
+## Command Reference
```

Comment:
The Command Reference should just refer to [src/cli/README.md](https://github.com/openthread/openthread/blob/master/src/cli/README.md)